tests/bluetooth/hci_prop_evt/prj.conf

```diff
@@ -4,7 +4,7 @@ CONFIG_ZTEST=y
 CONFIG_BT=y
 CONFIG_BT_CTLR=n
 CONFIG_BT_NO_DRIVER=y
-CONFIG_BT_RECV_IS_RX_THREAD=y
+CONFIG_BT_RECV_DIRECT=y
```

Comment:
Why do we need to set `CONFIG_BT_RECV_DIRECT=y` explicitly? Similar in `host_long_adv_recv`